Assets/Scripts/Enemy/EnemyView.cs

```diff
@@ -10,15 +11,38 @@ public class EnemyView : MonoBehaviour
 
         private EnemyController enemyController;
 
+        private int health;
```

Comment:
health for the enemy should be in the Model class and not stored in the View

Reply:
Done